scripts/optimize.py

```diff
@@ -65,7 +73,11 @@ def get_emission_limit():
 
     solver = "cbc"
 
-    emission_limit = get_emission_limit()
+    # get additional scalars, set to None at first
+    emission_limit = None
```

Comment:
for your information: `el_gas_relations` and `bpchp_out` will be added here in #102